tests/services/test_vision_processor.py

```
@@ -616,3 +617,134 @@ def test_context_manager(self, mock_vision_model: MagicMock) -> None:
+                generate_filename=False,
+                perform_ocr=False,
+            )
+
+        adaptive_logs = [
+            call
+            for call in mock_logger.debug.call_args_list
+            if call.args and "Adaptive vision timeout" in str(call.args[0])
+        ]
+        assert len(adaptive_logs) == 1
+        # Message should include the file name and the numeric timeout
+        msg, *args = adaptive_logs[0].args
```

Ruff (0.15.13)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)